src/main/java/picard/util/BedToIntervalList.java

```diff
@@ -107,9 +107,13 @@ public class BedToIntervalList extends CommandLineProgram {
     @Argument(doc = "If true, entries that are on contig-names that are missing from the provided dictionary will be dropped.")
     public boolean DROP_MISSING_CONTIGS = false;
 
+    @Argument(doc = "If true, write length zero intervals in input bed file to resulting interval list file.")
```

Comment:
Should we say "zero-length intervals" instead of "length zero intervals". I know it's not critical, just for the sake of consistency

Reply:
I actually like "length zero" better than "zero length" so I switched the labelings everywhere to make it consistent.